### DISCUSSION TITLE
Forbid direct use of process.env (import.meta.env)

### DISCUSSION THREAD

**Discussion by gregorjan** [biomejs/biome#3525] [Rule suggestion]
I would like to set up new rule to prevent direct usage of process.env similar to no-process-env eslint-plugin-n rule.

```
if(process.env.NODE_ENV === "development") {
    //...
}
```
This would result in an error.

```
import {env} from './env'

if(env.NODE_ENV === "development") {
    //...
}
```
Would not result in an error

**Comment by ematipico** [comment]
Please use this discussion to propose rules from other sources: https://github.com/biomejs/biome/discussions/3
